pep/app/utils/search.ts

```diff
@@ -72,16 +126,50 @@ export function buildSearchQueryParams(
         let facetType = SEARCH_FACETS.findBy('id', id);
         let facets = groupedFacets[id];
         if (facetType && facetType.param) {
+            const qp = queryParams[facetType.param];
             //join all the selected facet values together
             let facetValues = facets.mapBy('value').join(facetType.paramSeparator);
-            //if the query param already exists, join it to the existing one
-            if (queryParams[facetType.param]) {
-                queryParams[facetType.param] = `${queryParams[facetType.param]} AND ${facetValues}`;
-            } else {
-                queryParams[facetType.param] = facetValues;
+
+            if (facetType.prefixValues) {
+                facetValues = `${facetType.id}:(${facetValues})`;
             }
+
+            //if the query param already exists, join it to the existing one
+            queryParams[facetType.param] = `${qp ? `${qp} ${joinOp} ` : ''}${facetValues}`;
         }
     });
 
     return removeEmptyQueryParams(queryParams);
 }
+
+/**
+ * A "blank" search request could have the following params:
+ * synonyms, facetfields, and abstract, citecount, viewcount, viewperiod
+ * If there are params besides that, it is a valid search form submission
+ * @export
+ * @param {QueryParamsObj} params
+ * @returns
+ */
+export function hasSearchQuery(params: QueryParamsObj) {
+    const exclude = ['synonyms', 'facetfields', 'abstract', 'citecount', 'viewcount', 'viewperiod'];
```

Comment:
You might want to make it so you can pass in the exclude params and just default them that way it can be changed if it they need it to be down the line